lib/src/modules/task.cc

```diff
@@ -510,12 +510,14 @@ void Task::processOutputAndUpdateMetadata(ActionResponse& response)
     std::string &output = response.output.std_out;
 
     if (isValidUTF8(output)) {
+        // Use a temporary object to create a JSON string. The JsonContainer API doesn't
+        // provide a way to set the root to a string unless that string is already quoted.
         lth_jc::JsonContainer stdout_result;
-        stdout_result.set("output", output);
+        stdout_result.set("_output", output);
 
-        response.setValidResultsAndEnd(std::move(stdout_result));
+        response.setValidResultsAndEnd(stdout_result.get<lth_jc::JsonContainer>("_output"));
```

Comment:
Can we not use the `_output` key here, since the semantics of this key are very different from the orchestrator's `_output` key?

Reply:
It's completely omitted in the results, but I can use something that's obvious not a real key.